core/src/main/java/org/apache/accumulo/core/util/Halt.java

```diff
@@ -51,9 +50,10 @@ public void run() {
   public static void halt(final int status, Runnable runnable) {
     try {
       // give ourselves a little time to try and do something
-      new Daemon() {
+      new Thread() {
         @Override
         public void run() {
+          setDaemon(true);
```

Comment:
Is it legal to switch to a Daemon thread after it has already started executing? Will this have any effect?

Reply:
Great catch. This won't work, I will fix.